src/frontend/apps/lti_site/components/PublicVideoDashboard/PublicVODDashboard/index.tsx

```diff
@@ -37,15 +37,16 @@ export const PublicVODDashboard = ({ playerType }: PublicVODDashboardProps) => {
   }
 
   return (
-    <Box>
+    <Box background={{ color: 'bg-marsha' }}>
       <VideoPlayer
         video={video}
         playerType={playerType}
         timedTextTracks={timedTextTracks}
       />
-      {video.show_download && <DownloadVideo urls={video.urls} />}
-      {transcripts.length > 0 && (
-        <Transcripts transcripts={transcripts as TimedTextTranscript[]} />
+
+      {transcripts.length > 0 && <Transcripts transcripts={transcripts} />}
```

Comment:
Transcripts should become a widget and moved to the `VideoWidgetProvider` mechanism. Once made, only the VideoWidgetProvider should be mounted and thwe widget should be display or not in it.
Without doing this, it will be difficult to add new widgets.